omod/src/main/java/org/openmrs/module/orderextension/web/tag/FormatTag.java

```diff
@@ -34,7 +34,8 @@ public int doStartTag() {
 		} 
 		else {
 			try {
-				pageContext.getOut().write(result);
+				if(result != null)
+					pageContext.getOut().write(result);
```

Comment:
By convention, please add curly braces around your conditional - leaving them out often leads to unexpected errors down the road.  Thanks!